gledki_test.go

```diff
@@ -293,11 +295,10 @@ func TestFtExecString(t *testing.T) {
 func TestErrors(t *testing.T) {
 
 	if _, err := New([]string{"/ala/bala/nica"}, filesExt, tagsPair, false); err != nil {
-		errstr := err.Error()
-		if strings.Contains(errstr, "does not exist") {
-			t.Logf("Right error: %s", err.Error())
+		if errors.Is(err, os.ErrNotExist) {
+			t.Logf("Right error: %v", err)
 		} else {
-			t.Fatalf("Wrong error: errstr")
+			t.Fatalf("Wrong error: %v", err)
```

Comment:
I strongly suggest not to use `t.Fatal*`, unless you really want to abort the test. Better to use `t.Error*`.

- `t.Error*` -> t.Log + t.Fail, which marks the testcase as failed, but continues
- `t.Fatal*` -> t.Log + t.FailNow, which aborts the current test function

Same applies to the following test cases